tasks/skylab/StarAlign.wdl

```diff
@@ -130,3 +130,177 @@ task StarAlignFastqPairedEnd {
   }
 
 }
+
+
+task STARsoloFastq {
+  input {
+    Array[File] r1_fastq
+    Array[File] r2_fastq
+    File tar_star_reference
+    File white_list
+    String chemistry
+    String counting_mode
+
+    # runtime values
+    String docker = "quay.io/humancellatlas/secondary-analysis-star:v2.7.9a"
+    Int machine_mem_mb = 64000
+    #ceil((size(tar_star_reference, "Gi")) + 6) * 1100
+    Int cpu = 16
+    # multiply input size by 2.2 to account for output bam file + 20% overhead, add size of reference.
+    Int disk = ceil((size(tar_star_reference, "Gi") * 3)) + ceil(size(r1_fastq, "Gi") * 20) +  ceil(size(r2_fastq, "Gi") * 20)
+    # by default request non preemptible machine to make sure the slow star alignment step completes
+    Int preemptible = 0
```

Comment:
I really prefer defaulting to use preemptible UNLESS we **KNOW** this will typically take greater than 24 hours.  Since this is in the input section of the task, it can be overridden through the inputs in the inputs json. We can use this to always run non-preemptible for our production purposes and we can default to at least 1 preemptible for the pipeline released for the community

Reply:
Changed to 3.